api/src/main/resources/org/openmrs/module/drcreports/sql/DRCArtLessthanThreeMonthsDispense.sql

```diff
@@ -0,0 +1,25 @@
+SELECT DISTINCT p.patient_id
+FROM patient p
+INNER JOIN person pe ON p.patient_id = pe.person_id
+WHERE pe.voided = 0 
+  AND p.voided = 0
+  AND EXISTS (
+    SELECT 1
+    FROM obs o
+    INNER JOIN concept c ON o.concept_id = c.concept_id
+    WHERE o.person_id = p.patient_id
+      AND o.voided = 0
+      AND c.uuid = '3a0709e9-d7a8-44b9-9512-111db5ce3989'
+      AND o.value_numeric IS NOT NULL
+      AND o.value_numeric < 90
```

Comment:
Technically `o.value_numeric < 90` will also eliminate rows where `value_numeric` is null

Reply:
SQL uses a 3-value logic where things are `TRUE`, `FALSE` and `NULL` that uses something like these truth tables:

|Value|NOT Value|
|---|---|
|True|False|
|False|True|
|Null|Null|

|Value|AND True|AND False|AND Null|
|---|---|---|---|
|True|True|False|Null|
|False|False|False|Null|
|Null|Null|Null|Null|

|Value|OR True|OR False|OR Null|
|---|---|---|---|
|True|True|True|Null|
|False|True|False|Null|
|Null|Null|Null|Null|

Basically, any operation that involves a `NULL` results in a `NULL`, and `NULL` is not true, so the row isn't matched.